services/api/service.go

```diff
@@ -137,11 +137,13 @@ type RelayAPIOpts struct {
 }
 
 type payloadAttributesHelper struct {
-	slot              uint64
-	parentHash        string
-	withdrawalsRoot   phase0.Root
-	parentBeaconRoot  *phase0.Root
-	payloadAttributes beaconclient.PayloadAttributes
+	slot                   uint64
+	parentHash             string
+	withdrawalsRoot        phase0.Root
+	parentBeaconRoot       *phase0.Root
+	payloadAttributes      beaconclient.PayloadAttributes
+	depositReceiptsRoot    phase0.Root
```

Comment:
has https://github.com/attestantio/go-eth2-client/blob/master/api/v1/payloadattributesevent.go been updated for latest payload attributes?
If so it can replace the types https://github.com/flashbots/mev-boost-relay/blob/main/beaconclient/prod_beacon_instance.go#L63-L83 which need updating to accept the new payload attributes. You'll also need to update `processPayloadAttributes` in `service.go` with the new payload attributes